src/interpreter/__tests__/builtins.ts

```diff
@@ -0,0 +1,32 @@
+import { parse } from '../../parser/parser';
+import { compileProgram } from '../compiler';
+import { interpret } from '../virtualMachine';
+
+describe('built-ins', () => {
+  test('handles built-in functions', () => {
+    const code = `
+        int main() {
+            return sqrt(25);
```

Comment:
What happens if I do `sqrt(7)`, aka if the result is non int? Does it matter? I think it doesn't really matter for our proof of concept but just wondering if you have any thoughts?

Reply:
You get a float. This is true of everything in our current implementation by the way. The following is a valid program:
```c
int main() {
    return 7.5;
}
```

I'm guessing we handle when type checking?